plugin/cr/src/main/java/org/opencds/cqf/ruler/cr/KnowledgeArtifactProcessor.java

```diff
@@ -1712,6 +1726,24 @@ public MetadataResource revise(HapiFhirRepository hapiFhirRepository, MetadataRe
 
 		return resource;
 	}
+
+	private static Library getRootSpecificationLibrary(List<BundleEntryComponent> bundleEntries) {
+		Optional<Library> rootSpecLibrary = bundleEntries.stream()
+			.filter(entry -> entry.getResource().getResourceType().equals(ResourceType.Library))
```

Comment:
nitpick: `==` for enums instead of `.equals(`

Reply:
Fixed!